src/MarketingPageAcceptanceTests.SupplierTests/Features/AboutOrganisation/EditAboutSupplier.feature

```diff
@@ -30,19 +30,4 @@ Scenario: About Supplier Section - Marked as Complete -  Any Data Saved
 Scenario: About Supplier Section - Marked as Incomplete -  No Data
 	Given the About supplier section does not require Mandatory Data
 	When the Marketing Page Form is presented 
-	Then the About supplier section is marked as Incomplete
-
-Scenario: About Supplier Section - Pre-Populated Description & URL
-	Given the User has created a new solution for the same supplier (Solution B)
-	When the User is editing the About supplier section for Solution B
-	Then the data will be populated in the About supplier Section
-
-Scenario Outline: About Supplier Section - Changing About Supplier Data
-	Given the User has created a new solution for the same supplier (Solution B)
-	And the User is editing the About supplier section for Solution B
-	When the About Supplier <field> data is changed for Solution B 
-	Then the About Supplier data is changed for Solution A as well as for Solution B
-	Examples: 
-	| field       |
-	| description |
-	| link        |
\ No newline at end of file
+	Then the About supplier section is marked as Incomplete
```

Comment:
The old favourite: trailing new line. Do you have `insert_final_newline` set in your `.editorconfig`?

```
[*]
indent_style = space
insert_final_newline = true
```

Reply:
That is set in the editorconfig, but it doesn't seem to work very well on feature files